lib/Logger.js

```diff
@@ -61,7 +70,7 @@ class Logger {
         const rLog = new RequestLogger(Config.logger,
                                        Config.level, Config.dump, Config.end,
                                        uids);
-        rLog.addDefaultFields({ name: this.name });
+        rLog.setParent(this);
```

Comment:
setting a parent is only triggerring the "transmission" of the parent's `precomputedFields` to the child, thus computing the `precomputedFields` of the child (here the `rLog`).
As such, this is not much heavier than the previous methods.